step2_RelaxModel/make/makefile

```diff
@@ -21,7 +21,7 @@ endif
 # STAR_WORK_OBJS defined in makefile_header
 # added more code located in WORK_SRC_DIR
 IONIZATION_OBJS = ionization_def.o ion_tables_load.o mod_tables.o mod_ionization.o ion_tables_eval.o ion_table_plot.o ionization_lib.o
-INSPIRAL_OBJS = CE_orbit.o CE_energy.o CE_torque.o CE_adjust_mdot.o CE_before_struct_burn_mix.o CE_after_struct_burn_mix.o CE_timestep.o CE_run_star_extras.o
+INSPIRAL_OBJS = dop853_module.o  CE_orbit.o CE_energy.o CE_torque.o CE_adjust_mdot.o CE_before_struct_burn_mix.o CE_after_struct_burn_mix.o CE_timestep.o CE_run_star_extras.o 
```

Comment:
I'd suggest you to check the spaces: no double space between two object files, no space at the end of the line.